apps/platform-app/src/app/[locale]/(profile)/(components)/TogglePublishButton/TogglePublishButton.tsx

```diff
@@ -13,16 +13,25 @@ import { useTranslations } from 'next-intl'
 import { publishProfile } from '../../_actions/mutations/publishProfile'
 import { unpublishProfile } from '../../_actions/mutations/unpublishProfile'
 import { DevTypeButton } from '../Filters/Buttons/DevTypeButton/DevTypeButton'
+import VerificationModal from '../VerificationModal/VerificationModal'
 
 interface TogglePublishButtonProps {
   profileId: string
```

Comment:
**issue**: The `TogglePublishButton` component should not be async as it doesn't perform any asynchronous operations directly.